int-set/src/bitset.rs

```diff
@@ -11,28 +11,26 @@ const PAGE_BITS_LOG_2: u32 = PAGE_BITS.ilog2();
 
 /// An ordered integer (u32) set.
 #[derive(Clone, Debug)]
-pub(crate) struct BitSet<T> {
+pub(crate) struct BitSet {
     // TODO(garretrieger): consider a "small array" type instead of Vec.
     pages: Vec<BitPage>,
     page_map: Vec<PageInfo>,
     len: Cell<usize>, // TODO(garretrieger): use an option instead of a sentinel.
-    phantom: std::marker::PhantomData<T>,
 }
 
-impl<T: Into<u32> + Copy> BitSet<T> {
+impl BitSet {
```

Comment:
agree it makes sense to not have any trait bounds here, and let that live up above 👍